evap/staff/templates/staff_template_form.html

```diff
@@ -8,11 +10,24 @@
 
 {% block content %}
     {{ block.super }}
+
     <form id="template-form" method="POST" enctype="multipart/form-data" class="form-horizontal multiselect-form">
         {% csrf_token %}
         <div class="card mb-3">
-            <div class="card-body">
-                {% include 'bootstrap_form.html' with form=form %}
+            <div class="card-body row">
+                <div class="col-9">
+                    {% include 'bootstrap_form.html' with form=form email_form=True %}
+                </div>
+                <div class="col-3">
+                    <p>{% trans 'The following variables are available for this email template:' %}</p>
+                    {% for available_variable in available_variables %}
+                        <code>{{ available_variable }}</code>
+                        <button type="button" class="btn btn-sm btn-link" title="{% trans 'copy variable to clipboard' %}" onClick="copyToClipboard('{{ available_variable }}')">
+                            <i class="fas fa-clipboard"></i>
+                        </button>
+                        <br />
+                    {% endfor %}
```

Comment:
some changes i want to suggest:
- we usually use `span` for icons.
- i think the clipboard icon should be placed before the variables for a cleaner layout and should get some spacing. also, the lighter version seems to fit better in my opinion.
- the tooltip must be bootstrapyfied :)
```suggestion
                    {% for variable in available_variables %}
                        <button type="button" class="btn btn-sm btn-link text-dark pr-1" data-toggle="tooltip" data-placement="top" title="{% trans 'Copy to clipboard' %}" onClick="copyToClipboard('{{ variable }}')">
                            <span class="far fa-clipboard"></span>
                        </button>
                        <code>{{ variable }}</code>
                        <br />
                    {% endfor %}
```